server/text_generation_server/utils/flash_attn_triton.py

```diff
@@ -0,0 +1,816 @@
+#!/usr/bin/env python
```

Comment:
Shouldn't we put this in `layers/flash_attn/triton.py` maybe ? (and  `flash_attn.py`-> `flash_attn/__init__.py` for simplificity ?)

Reply:
Could we do that in an other PR? Then e.g. paged_attention.py should be moved as well